enhancements/cluster-logging/forwarder-to-otlp.md

```diff
@@ -0,0 +1,201 @@
+so that I can aggregate logs in my existing Red Hat manged log storage solution
+
+* As an administrator, I want to forward logs to an OTLP receiver
+so that I can use my observability tools to evaluate all my signals (e.g. logs, traces, metrics)
+
+### Goals
+
+* Implement OTLP over HTTP in the log collector to forward to any receiver that implements OTLP
+* Deprecate the [ViaQ Data Model](https://github.com/openshift/cluster-logging-operator/blob/release-5.9/docs/reference/datamodels/viaq/v1.adoc) in
+favor of [OpenTelemetry Semantic Conventions](https://opentelemetry.io/docs/specs/semconv/)
+* Allow customer to continue to use **ClusterLogForwarder** as-is in cases where they are not ready or not interested in OpenTelementry
+
+### Non-Goals
+
+* Replace the existing log collection agent with the [OTEL Collector](https://opentelemetry.io/docs/collector/)
+* Utilize OpenTelemetry Semantic Conventions when writing to other `Output` types
+
+## Proposal
+
+This section should explain what the proposal actually is. Enumerate
+*all* of the proposed changes at a *high level*, including all of the
+components that need to be modified and how they will be
+different. Include the reason for each choice in the design and
+implementation that is proposed here.
+
+To keep this section succinct, document the details like API field
+changes, new images, and other implementation details in the
+**Implementation Details** section and record the reasons for not
+choosing alternatives in the **Alternatives** section at the end of
+the document.
+
+This proposal inte
+
+### Workflow Description
+
+**cluster administrator** is a human responsible for administering the **cluster-logging-operator**
+and **ClusterLogForwarders**
+
+1. The cluster administrator deployes the cluster-logging-operator if it is already not deployed
+1. The cluster administrator edits or creates a **ClusterLogForwarder** and defines an OTLP output
+1. The cluster administrator references the OTLP output in a pipeline
+1. The cluster-logging-operator reconciles the **ClusterLogForwarder**, generates a new collector configuration,
+and updates the collector deployment
+
+
+### API Extensions
+
+```yaml
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+spec:
+outputs:
+- name:
+  type:           # add otlp to the enum
+  tls:
+    secret:             #the default resource to search for keys
+      name:                # the name of resource
+      cacert:
+        secret:               #enum: secret, configmap 
+          name:                # the name of resource
+        key:                 #the key in the resource
+      cert:
+        key:                 #the key in the resource
+      key:
+        key:                 #the key in the resource
+    insecureSkipVerify:
+    securityProfile: 
+  otlp:
+    url:                   #appends '/v1/logs' if endpoint is missing subpath
```

Comment:
I would suggest being more explicit and accepting the final URL

Reply:
How do you propose to identify what needs to be documented here?  I think the spec states endpoint MUST terminate with this suffix.  If a user gives us a base we can craft the URL.  Does that mean:
1. we should expect the correct URL and add validation
1. append the suffix to the URL if it doesn't have it?
1. try to pull the base info from what they give us and craft the url?

Sound maybe like you are suggesting "1".  This would actually fit into our flow that exists now.

Reply:
I also vote for 1

Reply:
I think the 1. makes the most sense

Reply:
updated to:

```  otlp:
    url:                   #must terminate with '/v1/logs'
```